src/utilities/HYPRE_utilities.h

```diff
@@ -122,9 +122,37 @@ typedef HYPRE_Int MPI_Comm;
  * HYPRE init/finalize
  *--------------------------------------------------------------------------*/
 
-HYPRE_Int HYPRE_Init(void);
+/**
+ * Initializes the hypre library.
+ **/
+
+HYPRE_Int HYPRE_Initialize(void);
+
+/**
+ * Initializes the hypre library. This function is provided for backwards compatibility.
+ * Please, use HYPRE_Initialize instead.
+ **/
+
+#define HYPRE_Init(...) HYPRE_DEPRECATED("GCC warning \"Use HYPRE_Initialize() instead.\"") HYPRE_Initialize(__VA_ARGS__)
```

Comment:
Is `HYPRE_Init` still supported?  Also, is `...` part of the C standard we currently support?

Reply:
Hi Rob, yes, it's supported but if someone tries to use it, there will be a compiler warning. Variadic macros `(...)` are C99, but it's actually not needed in this case because `HYPRE_Init` has no input arguments, I'll update!

Reply:
Okay, now that I've had a bit more time to stare at this, I see what's happening.  Thanks!